dotnetv3/Comprehend/README.md

```diff
@@ -41,11 +41,12 @@ Code excerpts that show you how to call individual service functions.
 - [DetectSyntax](DetectingSyntaxExample/DetectingSyntaxExample/DetectingSyntax.cs#L6)
 - [StartTopicsDetectionJob](TopicModelingExample/TopicModelingExample/TopicModeling.cs#L6)
 
-### Cross-service examples
+### Scenarios
 
-Sample applications that work across multiple AWS services.
+Code examples that show you how to accomplish a specific task by calling multiple
+functions within the same service.
 
-- [Create an application to analyze customer feedback](../cross-service/FeedbackSentimentAnalyzer)
+- [Create an application to analyze customer feedback](dotnetv3/cross-service/FeedbackSentimentAnalyzer)
```

Comment:
Check this link, and I think it's happening to all of the scenarios relative links.

Reply:
Fixed